core/src/ConfirmationModal.html

```diff
@@ -95,6 +95,7 @@ <h2 class="fd-title fd-title--h5">{settings.header}</h2>
   @import 'styles/variables';
 
   .fd-message-box {
+    z-index:1001;
```

Comment:
We should manage all z-index in a central file. It's fine to do it in a separate ticket.
https://www.smashingmagazine.com/2021/02/css-z-index-large-projects/

Reply:
Let's discuss it after daily and there is another z-index: 1100; in our code for alert overlay (.luigi-alert--overlay)

Reply:
👍  jup let's discuss in the daily.. but I like the idea to manage the z-index in a central file